src/controller/OperationalCredentialsDelegate.h

```diff
@@ -76,6 +76,12 @@ class DLL_EXPORT OperationalCredentialsDelegate
      *   fabric ID.
      */
     virtual void SetFabricIdForNextNOCRequest(FabricId fabricId) {}
+
+    virtual CHIP_ERROR GenerateNOCSR(MutableByteSpan & csrNonce)
```

Comment:
This should be renamed to `ObtainCsrNonce`

Reply:
Addressed in latest commit